certified-connectors/Maximizer/apiDefinition.swagger.json

```diff
@@ -0,0 +1,5059 @@
+                      "value": 3
+                    }
+                  ]
+                },
+                "sort": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Sort by",
+                  "enum": ["Creation Date", "Last Modify Date"]
+                },
+                "order": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Sort Order",
+                  "enum": ["DESC", "ASC"]
+                },
+                "limit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "Search Limit"
+                },
+                "single": {
+                  "type": "boolean",
+                  "description": "Apply only when the search criteria returns a single record",
+                  "title": "Restrict single record"
+                },
+                "udf": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "x-ms-dynamic-list": {
+                      "operationId": "DropdownLeadUdf",
+                      "itemValuePath": "Key",
+                      "itemsPath": "Values",
+                      "itemTitlePath": "DisplayValue"
+                    }
+                  },
+                  "description": "Select User Defined Fields (UDFs) to include",
+                  "title": "UDFs"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/api/DropdownLeadProcess": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK, Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Code": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Code"
+                },
+                "State": {
+                  "type": "string",
+                  "description": "State"
+                },
+                "Values": {
+                  "type": "array",
+                  "description": "Values",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "sample": {
+                        "type": "string",
+                        "description": "sample"
+                      },
+                      "value": {
+                        "type": "string",
+                        "description": "value"
+                      },
+                      "label": {
+                        "type": "string",
+                        "description": "label"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "DropdownLeadProcess",

```

Comment:
@MaximizerIntegration all operations' summaries should be a sentence and different than the description. please see our [docs](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations) for more information.

Please fix this for all the operations